streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImpl.java

```diff
@@ -46,7 +46,12 @@
 
     private final Deque<String> threadLevelSensors = new LinkedList<>();
     private final Map<String, Deque<String>> taskLevelSensors = new HashMap<>();
+    private final Map<String, Deque<String>> nodeLevelSensors = new HashMap<>();
     private final Map<String, Deque<String>> cacheLevelSensors = new HashMap<>();
+    private final Map<String, Deque<String>> storeLevelSensors = new HashMap<>();
+
+    private static final String SENSOR_PREFIX_DELIMITER = ".";
+    private static final String SENSOR_NAME_DELIMITER = ".s.";
```

Comment:
I've made the following attempts to codify the sensor names:
* moved the delimiters here
* created a method for each level's prefix.
* (re)created methods for the external parent and child sensor names
* namespaced the internal sensors "internal" and the external sensors "external"